src/components/pages/docs.js

```diff
@@ -206,16 +203,20 @@ function Docs({ initialState, PageTemplate }) {
             <LikeOutlined className="u-verified-badge" />
           </Tooltip>
         )}
-        <Tooltip title={t(`common:allowedOpenContributionBadge_${row.allowedOpenContribution}`)}>
-          <div className="u-allowed-open-contribution-badge">
-            {row.allowedOpenContribution === DOCUMENT_ALLOWED_OPEN_CONTRIBUTION.none
-              && <AllowedOpenContributionNoneIcon />}
-            {row.allowedOpenContribution === DOCUMENT_ALLOWED_OPEN_CONTRIBUTION.content
-              && <AllowedOpenContributionContentIcon />}
-            {row.allowedOpenContribution === DOCUMENT_ALLOWED_OPEN_CONTRIBUTION.metadataAndContent
-              && <AllowedOpenContributionMetadataAndContentIcon />}
-          </div>
-        </Tooltip>
+        {row.allowedOpenContribution === DOCUMENT_ALLOWED_OPEN_CONTRIBUTION.content && (
+          <Tooltip title={t('allowedOpenContributionBadge_content')}>
+            <div className="u-badge">
+              C
+            </div>
+          </Tooltip>
+        )}
+        {row.allowedOpenContribution === DOCUMENT_ALLOWED_OPEN_CONTRIBUTION.metadataAndContent && (
+          <Tooltip title={t('allowedOpenContributionBadge_metadataAndContent')}>
+            <div className="u-badge">
```

Comment:
I was wondering if we should set `user-select: none;` on the `u-badge` class (or maybe set the cursor to question mark shape), because (at least in this use case) the combination of the caret shaped cursor and the tooltip looks not as nice.

Similarly the language "badge" should maybe also not have the caret (and maybe a tooltip showing the actual language name in full text in the current UI language).

What do you think?

Reply:
True, I hadn't notice the caret cursor. Super good idea also with the language 👏 
Changed all as suggested.